### DISCUSSION TITLE
cannot insstall deta on windowss 1 with powersshel

### DISCUSSION THREAD

**Discussion by worldsoft998** [deta/cloud-docs#733] [Help]
Output:
===========
Invoke-WebRequest : 无法连接到远程服务器 cannot connect to the remote server
所在位置 行:62 字符: 1 line 62  character 1
+ Invoke-WebRequest $DetaUri -OutFile $DetaZip -UseBasicParsing
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest]，WebExceptio 
   n
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
 
My powershell:
============
PSVersion                      5.1.19041.2364                                                                              
PSEdition                      Desktop                                                                                     
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                     
BuildVersion                   10.0.19041.2364                                                                             
CLRVersion                     4.0.30319.42000                                                                             
WSManStackVersion              3.0                                                                                         
PSRemotingProtocolVersion      2.3                                                                                         
SerializationVersion           1.1.0.1

**Comment by jnsougata** [comment]
Can you please try with powershell `version >= v6.0.0`?
